code/modules/hydroponics/seed_datums.dm

```diff
@@ -189,15 +189,14 @@
 	name = "bloodtomato"
 	seed_name = "blood tomato"
 	display_name = "blood tomato plant"
-	mutants = list("killertomato")
```

Comment:
I'd keep it as a potential mutation of blood tomatoes, personally. Giving normal tomatoes the chance to mutate into them is fine though